packages/auth/src/grant/service.ts

```diff
@@ -16,8 +16,12 @@ import { AccessService } from '../access/service'
 import { Pagination } from '../shared/baseModel'
 import { FilterString } from '../shared/filters'
 
+interface FilterGrantState {
+  notIn?: GrantState[]
+}
 interface GrantFilter {
   identifier?: FilterString
+  state?: FilterGrantState
 }
 
 export interface GrantService {
```

Comment:
I found the following unused service methods when auditing the reads for access, access tokens, and grants:

- `GrantService.get`
- `GrantService.getByInteraction`
- `AccessTokenService.get`

I left them to be safe but wonder if maybe we should remove them.

Reply:
`GrantService.getByInteraction` we can probably get rid of, I believe it has been supplanted by `GrantService.getByInteractionSession`. We could also remove `(GrantService/AccessTokenService).get` too, I wrote those anticipating they would be used but as I wrote out the rest of the GNAP implementation they didn't end up getting used.